ldap/servers/plugins/replication/repl5.h

```diff
@@ -710,6 +711,8 @@ PRBool replica_is_updatedn(Replica *r, const Slapi_DN *sdn);
 void replica_set_updatedn(Replica *r, const Slapi_ValueSet *vs, int mod_op);
 void replica_set_groupdn(Replica *r, const Slapi_ValueSet *vs, int mod_op);
 char *replica_get_generation(const Replica *r);
+bool replica_check_validity(Replica *replica);
```

Comment:
It feels a bit inconsistent to use `bool` while every other line in the file uses `PRBool`... But I understand we might get rid of `PRBool` at some point.

I don't have a hard opinion here, but generally, I'd prefer consistency in this case (my 5 cents). :)

Reply:
FYI: I discussed that one with Mark in another review, and finally we decided to use stdbool in new code.
I agree it is  inconsistent, until we do a global cleanup phase (but frankly I am afraid that there are dependencies toward the plugin API and we will never be able to complete that move )